src/node-lib/curriculum-api-2023/queries/pupilLesson/pupilLesson.schema.ts

```diff
@@ -42,7 +42,8 @@ export type ImageOrTextItem = ImageItem | TextItem;
 
 export const lessonContentSchema = lessonContentSchemaFull.omit({
   state: true,
-  cohort: true,
+  // cohort has type never, investigate.
+  // cohort: true,
```

Comment:
@benprotheroe do you know if this is required? Can we remove this line?

Reply:
yeah i dont think you need this line, lessonContentSchemaFull doesnt have a cohort field